README.md

```diff
@@ -11,6 +11,9 @@ A Pivotal Tracker API gem that can be used to interface with the Pivotal Tracker
 ```ruby
 # Use your personal pivotal token
 PivotalAPI::Service.set_token A_PIVOTAL_TOKEN
+
+#then get me object
```

Comment:
Can you add a space between `#` and `then`? Super small, I know. But would like to keep consistency with other comments. Thanks.